.github/workflows/build-and-runtime-test.yml

```diff
@@ -106,7 +115,7 @@ jobs:
         uses: actions/upload-artifact@v3
         if: ${{ failure() && steps.e2e.outcome != 'success' }}
         with:
-          name: canary-cypress-error-${{ matrix.path }}
+          name: canary-cypress-error-${{ matrix.framework }}-${{ matrix.build-tool }}
```

Comment:
Oh I see, this is where the artifact name is fixed

Reply:
yep